plugins/Toolbox/resources/qml/ToolboxDetailList.qml

```diff
@@ -6,19 +6,20 @@ import QtQuick.Controls 1.4
 import QtQuick.Controls.Styles 1.4
 import UM 1.1 as UM
 
-Item
+Rectangle
 {
-    id: base
+    id: detailList
+    // color: "green"
     ScrollView
     {
         frameVisible: false
-        anchors.fill: base
+        anchors.fill: detailList
         style: UM.Theme.styles.scrollview
         Column
         {
             anchors
             {
-                right: base.right
+                right: detailList.right
```

Comment:
QML says: "Cannot anchor to an item that isn't a parent or sibling". It must be `parent.right`

Reply:
Fixed in 62aeb6c